dev-client/src/screens/ProjectSitesScreen.tsx

```diff
@@ -103,34 +86,42 @@ const SiteMenu = ({site}: SiteProps) => {
 
   return (
     <Menu
-      closeOnSelect={true}
-      trigger={triggerProps => (
-        <IconButton name="more-vert" {...triggerProps} />
-      )}>
+      contentStyle={{
+        backgroundColor: theme.colors.background.default,
+      }}
+      visible={visible}
+      onDismiss={closeMenu}
+      anchor={<IconButton onPress={openMenu} name="more-vert" />}>
       <ConfirmModal
-        trigger={onOpen => (
-          <SiteMenuItem
-            iconName="remove"
-            text={t('projects.sites.remove_site')}
-            onPress={onOpen}
-          />
-        )}
+        trigger={onOpen => {
+          return (
+            <Menu.Item
+              title={t('projects.sites.remove_site')}
+              onPress={onOpen}
+              titleStyle={MENU_ITEM_STYLE}
+            />
+          );
+        }}
         title={t('projects.sites.remove_site_modal.title')}
-        body={t('projects.sites.remove_site_modal.body', {siteName: site.name})}
+        body={t('projects.sites.remove_site_modal.body', {
+          siteName: site.name,
+        })}
         actionName={t('projects.sites.remove_site_modal.action_name')}
         handleConfirm={removeSiteFromProjectCallback}
       />
 
       <ConfirmModal
         trigger={onOpen => (
-          <SiteMenuItem
-            iconName="delete"
```

Comment:
Have we confirmed that it's okay to omit the icons here? The figma designs still show them.

Reply:
Aha. The issue I had was to remove the `.`, which is what the icons appeared as. @CourtneyLee333 okay for this to be a regular menu w/o icons?